src/api/get.js

```diff
@@ -0,0 +1,39 @@
+const URL = 'http://localhost:3000'
+
+const apiHelper = async (endpoint, ...restParams) => {
+  const urlParams = restParams
+    .slice(0)
+    .unshift('/')
+    .join('/')
```

Comment:
If there are other params passed in, I want to join them together in a string where they are separated by slashes.

`slice(0)` makes a copy of the array. Because `unshift` mutates the array rather than returning a copy, I force a copy to be made. You always want to avoid modifying variables that are passed into functions as parameters.
`unshift('/')` adds a slash to the beginning of the array so that the slash will be added. Typing this out now, I realize that this plus the `join('/')` statement below will create a double slash at the beginning, so I will need to fix that.
`join('/')` brings all the elements in an array together in a string with a `/` between them.